layouts/default.vue

```diff
@@ -0,0 +1,8 @@
+<script setup lang="ts"></script>
+<template>
+	<div>
+		<TheHeader />
+		<slot />
+		<TheFooter />
+	</div>
+</template>
```

Comment:
Are those root elements still required in Nuxt?

```suggestion
<template>
	<TheHeader />
	<slot />
	<TheFooter />
</template>
```